### PR TITLE
Add pom files for scenario 5, sub scenario 5.1 and approch 5.1.1

### PR DESCRIPTION
**Pull request by RidmiR**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: product-scenarios/5-Route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/scenario_5.1-test-artifacts/scenario_5_1-synapse-config/src/main/synapse-config/proxy-services/1_1_2_soap_to_json_using_message_type.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <outSequence>
+            <property name="messageType" scope="axis2" type="STRING" value="application/xml"/>
+            <respond description="Respond to client"/>
+        </outSequence>
+        <faultSequence>
+            <makefault version="soap11">
+                <code expression="get-property('ERROR_CODE')" xmlns:soap11Env="http://schemas.xmlsoap.org/soap/envelope/"/>
+                <reason expression="get-property('ERROR_MESSAGE')"/>
+            </makefault>
+            <header action="remove" name="To" scope="default"/>
```
[the remark this answers is not in the document]
Comment (RidmiR) [reply product-scenarios/5-Route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/scenario_5.1-test-artifacts/scenario_5_1-synapse-config/src/main/synapse-config/proxy-services/1_1_2_soap_to_json_using_message_type.xml]: These test artifacts Files are mistakenly added to this PR. Will commit new changes.

File: product-scenarios/5-Route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/scenario_5.1-test-artifacts/scenario_5_1-synapse-config/src/main/synapse-config/proxy-services/1_1_2_soap_to_json_using_message_type.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy name="1_1_2_soap_to_json_using_message_type" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
```
[the remark this answers is not in the document]
Comment (RidmiR) [reply product-scenarios/5-Route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/scenario_5.1-test-artifacts/scenario_5_1-synapse-config/src/main/synapse-config/proxy-services/1_1_2_soap_to_json_using_message_type.xml]: These test artifacts Files are mistakenly added to this PR. Will commit new changes.

File: product-scenarios/5-Route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/scenario_5.1-test-artifacts/scenario_5_1-synapse-configCompositeApplication/pom.xml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.ei.product-scenarios.scenario_1_1</groupId>
+    <artifactId>scenario_1_1-synapse-configCompositeApplication</artifactId>
```
[the remark this answers is not in the document]
Comment (RidmiR) [reply product-scenarios/5-Route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/scenario_5.1-test-artifacts/scenario_5_1-synapse-configCompositeApplication/pom.xml]: These test artifacts Files are mistakenly added to this PR. Will commit new changes.

File: product-scenarios/5-Route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/5.1.1-Route-messages-based-on-the-given-xpath/5.1.1.1-Using-switch-mediator-on-given-xpath/pom.xml [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>5.1.1.1-Using-switch-mediator-on-given-xpath</artifactId>
+    <name>Route messages between system based on Xpath by using Switch Mediator</name>
+    <packaging>jar</packaging>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.12.4</version>
```
Comment (milindaperera) [inline product-scenarios/5-Route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/5.1.1-Route-messages-based-on-the-given-xpath/5.1.1.1-Using-switch-mediator-on-given-xpath/pom.xml]: let's remove version

**Comment by RidmiR** [comment]
@milindaperera Closing this PR as it needs to be rearranged with the new structure changes.

**Pull request closed without merging**